app/javascript/controllers/scroller_controller.js

```diff
@@ -0,0 +1,8 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  connect() {
+    this.element.scrollIntoView()
+    this.element.remove()
+  }
+}
```

Comment:
Is this controller used? I don't see a reference to it anywhere.

Reply:
Removed